wesnoth-httt.po

```diff
@@ -8725,17 +8592,13 @@ msgstr ""
 #. [message]: speaker=Delfador
 #: data/campaigns/Heir_To_The_Throne/scenarios/24_Battle_for_Wesnoth.cfg:399
 msgid "How dare I? I, High Provost of the Council of Archmagi..."
-msgstr "Jak se opovažuji? Jsem Delfador, Vysoký představený rady arcimágů..."
+msgstr "Jak se opovažuji? Jsem Delfador, Nejvyšší představený rady arcimágů..."
```

Comment:
Díky za stromového pána a další fajn opravy. Tady bych také dal malé. Jo, kdyby to byl Delfador Nejvyšší, pak by bylo velké na místě.
```suggestion
msgstr "Jak se opovažuji? Jsem Delfador, nejvyšší představený rady arcimágů..."
```